version/version.go

```diff
@@ -38,7 +38,7 @@ var Version string
 var HelperVersionOverride string
 
 // K3sVersion should contain the latest version tag of k3s (hardcoded at build time)
-var K3sVersion = "v1.21.7-k3s1"
+var K3sVersion = "v1.32.5+k3s1"
```

Comment:
```suggestion
var K3sVersion = "v1.32.5-k3s1"
```
It's the container image tag which does not support +

Reply:
Damn, sorry for the oversight and thanks for catching it!